### PR TITLE
Fluid json has changed

### PR DESCRIPTION
**Pull request by jginternational**
[FSI].[Fluid] Now ProjectParameters.json processes are inside an object, since KratosMultiphysics/GiDInterface#438 , resulting:

```json
"processes"            : {
        "initial_conditions_process_list"  : [],
        "boundary_conditions_process_list" : [{
            "python_module" : "apply_inlet_process",
            "kratos_module" : "KratosMultiphysics.FluidDynamicsApplication",
            "Parameters"    : {
                "model_part_name" : "AutomaticInlet2D_Inlet",
                "variable_name"   : "VELOCITY",
                "modulus"         : "6*y*(1-y)*sin(pi*t*0.5)",
                "direction"       : "automatic_inwards_normal",
                "interval"        : [0,1]
            }
        },{
            "python_module" : "apply_inlet_process",
            "kratos_module" : "KratosMultiphysics.FluidDynamicsApplication",
            "Parameters"    : {
                "model_part_name" : "AutomaticInlet2D_Inlet",
                "variable_name"   : "VELOCITY",
                "modulus"         : "6*y*(1-y)",
                "direction"       : "automatic_inwards_normal",
                "interval"        : [1,"End"]
            }
        }],
        "gravity"                          : [{
            "python_module" : "assign_vector_by_direction_process",
            "kratos_module" : "KratosMultiphysics",
            "process_name"  : "AssignVectorByDirectionProcess",
            "Parameters"    : {
                "model_part_name" : "Parts_Fluid",
                "variable_name"   : "BODY_FORCE",
                "modulus"         : 0.0,
                "constrained"     : false,
                "direction"       : [0.0,-1.0,0.0]
            }
        }],
        "auxiliar_process_list"            : []
    }

```

Notice I've changed it manually in the fsi script. Maybe you should try to let the Fluid app scripts make the read of the fluid part

Caution, if you had tests, this may break them

### CODE REVIEW

**Comment by loumalouomega** [comment]
@jginternational You are breaking how everything works, this is not "baladi" change. Blocking

**Review by loumalouomega — CHANGES_REQUESTED** [review]
@philbucher and @KratosMultiphysics/technical-committee, which is the way to proceed?

**Comment by philbucher** [comment]
This requires review from @rubenzorrilla , that why I would have also waited with merging KratosMultiphysics/GiDInterface#438

Same for the one in the Structural, I think @rubenzorrilla should first have a look

**Comment by jginternational** [comment]
@loumalouomega  That's why @rubenzorrilla was set as reviewer

**Comment by loumalouomega** [comment]
You are changing an analysis file, the design of the analysis took months, so the change is not trivial

**Comment by jginternational** [comment]
@loumalouomega That's why @rubenzorrilla was set as reviewer (to check it)

PD: Since FluidDynamics changed all the apps that depend on it, like fsi, potential fluid, embedded, are generating json files in the new format, and not working

**Comment by jginternational** [comment]
Ping @rubenzorrilla

**Comment by rubenzorrilla** [comment]
@jginternational  I've already updated this in the branch I told you this morning. It will be fully operative as soon as  #2417 is merged. I'm closing this since we've fixed much more issues in hotfix-FSI-problem-emulator-test.

**Pull request closed without merging**
